bazel/README.md

```diff
@@ -413,14 +413,12 @@ local_repository(
 
 ## Stats Tunables
 
-The default maximum number of stats in shared memory, and the default
-maximum length of a cluster/route config/listener name, can be
-overridden at compile-time by defining `ENVOY_DEFAULT_MAX_STATS` and
-`ENVOY_DEFAULT_MAX_OBJ_NAME_LENGTH`, respectively, to the desired
-value. For example:
+The default maximum length of a cluster/route config/listener name can be
+overridden at compile-time by defining `ENVOY_DEFAULT_MAX_OBJ_NAME_LENGTH`
```

Comment:
Is `ENVOY_DEFAULT_MAX_OBJ_NAME_LENGTH` used anymore either? I think all the truncation code is gone and this entire chain can be killed including deprecating the related CLI option?

Reply:
Done. This entailed some pretty far reaching changes, because StatsOptions now no longer has a reason to exist. The changes are almost all mundane refactoring, except RawStatData now has a hardcoded max name length (255; hopefully that's larger than anyone is using?)

Reply:
(RawStatData is now gone)